api/backend/ecs/load_balancer_manager.go

```diff
@@ -304,17 +303,23 @@ func (e *ECSLoadBalancerManager) createLoadBalancer(
 		return err
 	}
 
-	securityGroup, err := e.upsertSecurityGroup(ecsLoadBalancerID, ports)
-	if err != nil {
-		return err
+	// only public load balancers get an additional security group
```

Comment:
> I also spun up a consul cluster behind a private lb and it was able to communicate to itself as expected.

the main case I'm worried about here is a consul cluster in it's own environment, with a separate environment's service(s) trying to hit the consul cluster's private load balancer. If the private load balancer doesn't get the 'Access all' SG, I'm unclear on how other envs will be able to hit it.